bin/worker.js

```diff
@@ -0,0 +1,14 @@
+import { reportMetric } from "../lib/influx.js";
+
+export default {
+  async fetch(request, env, ctx) {
+    const response = await fetch(request);
+    ctx.waitUntil(
+      caches.default.put(request, response.clone())
+    )
```

Comment:
Is this line needed at all? Doesn't `fetch(request)` automatically automatically update the cache? Wouldn't `cache.default.put` create a cache record that will bypass further invocations of our worker? My questions are genuine; I don't know the answers myself.

Reply:
I can't find that in the docs. I've taken that at face value based on your [comment](https://github.com/filecoin-station/cf-metrics/pull/1#discussion_r1881546043). I have removed it since I think it's not needed, but testing it manually would actually be the best way to find out.